src/model/learn.py

```diff
@@ -7,9 +7,9 @@
 from stable_baselines3.common.vec_env import SubprocVecEnv
 
 from model.dqn import MaskedDQN, MaskedPolicy
+from model.ppo import MaskablePPO
 from environment.environment import WoWSimsEnv
 
-
```

Comment:
fyi in python 2 blank lines between imports and first line of source code is the standard, I'd just run `make lint`

Reply:
Thanks for the tip. I ran lint and it fixed up the formatting. It caught a few things in unchanged files, which I elected to leave alone for the moment. I also had to add a couple of dependencies to requirements for lint to run.